lib/OrganizationsFieldArray/SelectOrganizationCard.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  Layout
+} from '@folio/stripes/components';
+import css from './OrganizationCard.css';
+
+export default class SelectOrganizationCard extends React.Component {
+  static propTypes = {
+    headerEnd: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.arrayOf(PropTypes.node),
+    ]),
+    headerStart: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.arrayOf(PropTypes.node),
+    ]),
+    meta: PropTypes.shape({
+      error: PropTypes.node,
+    }),
+    renderEmpty: PropTypes.func,
+  };
+
+  renderError = (error) => (
+    <Layout className={`textCentered ${css.error}`}>
+      <strong>
+        {error}
+      </strong>
+    </Layout>
+  );
+
+  render() {
+    const {
+      headerEnd,
+      headerStart,
+      renderEmpty,
+    } = this.props;
+
+    const { meta } = this.props;
+    const error = meta && meta.touched && meta.error;
+    return (
+      <div>
+        <Card
+          cardStyle="negative"
+          data-test-organization-card
+          hasMargin
+          headerStart={headerStart}
+          headerEnd={headerEnd}
+          roundedBorder
+        >
+          { error ? this.renderError(error) : (renderEmpty && renderEmpty())}
```

Comment:
It's up to you, but would it make more sense to just use `this.props.children` instead of `renderEmpty` for this kind of thing? You could always render `this.props.children` to make sure any instructions in there are visible even when there's an error as well?